inc/classes/optimization/class-remove-query-string.php

```diff
@@ -87,76 +86,79 @@ public function __construct( Options $options, $busting_path, $busting_url ) {
 	}
 
 	/**
-	 * Set DOM crawler from provided HTML content
+	 * Remove query strings for CSS files that have one
 	 *
 	 * @since 3.1
 	 * @author Remy Perona
 	 *
-	 * @param HtmlPageCrawler $crawler Crawler instance.
-	 * @param string          $html   HTML content.
-	 * @return void
+	 * @param string $html HTML content.
+	 * @return string
 	 */
-	public function set_crawler( HtmlPageCrawler $crawler, $html ) {
-		$this->crawler = $crawler::create( $html );
+	public function remove_query_strings_css( $html ) {
+		$styles = $this->find( '<link\s+([^>]+[\s\'\"])?href\s*=\s*[\'\"]\s*?([^\'\"]+\.css(?:\?[^\'\"]*)?)\s*?[\'\"]([^>]+)?\/?>', $html );
```

Comment:
No need to escape double quotes: `[\s\'\"]`, `[\'\"]`, `[^\'\"]` => `[\s"\']`, `["\']`, `[^"\']`.